x-pack/qa/evil-tests/src/test/java/org/elasticsearch/xpack/security/authc/kerberos/SpnegoClient.java

```diff
@@ -62,14 +63,17 @@
      * Creates SpengoClient to interact with given service principal<br>
      * Use {@link #close()} to logout {@link LoginContext} and dispose
      * {@link GSSContext} after usage.
+     *
      * @param userPrincipalName User principal name for login as client
      * @param password password for client
      * @param servicePrincipalName Service principal name with whom this client
-     *            interacts with.
+     * interacts with.
+     * @param mech the Oid of the desired mechanism. Use (Oid) null to request
+     * the default mechanism.
      * @throws PrivilegedActionException when privileged action threw exception
      * @throws GSSException thrown when GSS API error occurs
      */
-    SpnegoClient(final String userPrincipalName, final SecureString password, final String servicePrincipalName)
+    SpnegoClient(final String userPrincipalName, final SecureString password, final String servicePrincipalName,final Oid mech)
```

Comment:
Needs a space, and we prefer more verbose variable names: 
```suggestion
    SpnegoClient(final String userPrincipalName, final SecureString password, final String servicePrincipalName, final Oid mechanism)
```